plugin.yaml

```diff
@@ -132,6 +140,30 @@ node_types:
         poststart:
           implementation: starlingx.cloudify_starlingx.resources.wrcp.poststart
 
+  cloudify.nodes.starlingx.PATCHES_LIST:
```

Comment:
We should use camel case on the final element in a node type or relationship type. The reason that `cloudify.nodes.starlingx.WRCP` is not camel case is because it is an abbreviation. But `PATCHES_LIST` can be `PatchList`.

Reply:
Fixed.